honeybadgermpc/reed_solomon.py

```diff
@@ -141,36 +147,29 @@ def __init__(self, d, point):
     def robust_decode(self, z, encoded):
```

Comment:
wasn't a fan of how there was a lot of shared logic, so hopefully this helps to clean up the control flow.

Reply:
Definitely much easier to read now